incubator/virtualcluster/config/setup/all_in_one.yaml

```diff
@@ -238,6 +238,21 @@ spec:
           image: virtualcluster/syncer-amd64
           imagePullPolicy: Always
           name: vc-syncer
+          env:
+          - name: STDLN_OBJ_ANNO_PREFXS
```

Comment:
how about `STDIN_OBJ_ANNO_PREFIXES` ?

PREFXS  this abbreviation is confusing

Reply:
change to `STDLN_OBJ_ANNO_PREFIXES`